s3-cloudfront-spa-tf/example-pattern.json

```diff
@@ -0,0 +1,64 @@
+{
+  "title": "AWS CloudFront and Amazon S3 for Single Page Application Hosting",
+  "description": "This project demonstrates hosting a Single-Page App using AWS CloudFront & S3, where CloudFront serves the SPA from an S3 bucket for static hosting.",
+  "language": "HCL",
```

Comment:
```suggestion
  "language": "Terraform",
```

Reply:
I know this seems incorrect but this is just how it is represented internally

Reply:
I have updated to Terraform